sandbox/src/routes/retrieveClinicalInformation.js

```diff
@@ -0,0 +1,39 @@
+const mockResponseProvider = require('../services/mockResponseProvider')
+const businessFunctionValidator = require('../services/businessFunctionValidator')
+
+module.exports = [
+  /**
+   * Sandbox implementation for retrieveClinicalInformation endpoint
+   */
+  {
+    method: 'POST',
+    path: '/STU3/v1/ReferralRequest/{ubrn}/$ers.generateCRI',
+    handler: (request, h) => {
+
+      const allowedBusinessFunctions = ["REFERRING_CLINICIAN", "REFERRING_CLINICIAN_ADMIN", "SERVICE_PROVIDER_CLINICIAN"]
+
+      const validationResult = businessFunctionValidator.validateBusinessFunction(request, h, allowedBusinessFunctions)
+      if (validationResult) {
+        return validationResult
+      }
+
+      // Simply checking if ubrn is supplied
+      if (request.params.ubrn) {
+
+        const { responsePath, responseCode, filename } = mockResponseProvider.getExampleResponseForRetrieveClinicalInformation();
+        if (responsePath !== undefined && responseCode !== undefined) {
```

Comment:
I think this could be simplified slightly to
`if (responsePath && responseCode)`

Reply:
I'll update it and alos improve all other sandbox endpoints that do make assertions like this